openapiart/bundler.py

```diff
@@ -336,8 +345,15 @@ def _validate_names(self, regex, components_key, components):
                             "%s property name `%s` contains invalid characters"
                             % (key, name)
                         )
+                    upper_case = self._check_upper_case(name)
+                    if upper_case:
+                        raise NameError("*** Property name '{}' is invalid. Only lower case with underscore separated is allowed.***".format(
```

Comment:
Only lower case with underscore separated is allowed -> Only lower case letters separated with an underscore is allowed

Reply:
done